### PR TITLE
ci: the release-v3.10 branch needs Kubernetes 1.28 jobs

### PR DESCRIPTION
**Pull request by nixpanic**
With Ceph-CSI v3.10 we do not test Kubernetes 1.25 anymore, and we
expect Kubernetes 1.28 to be used.

Signed-off-by: Niels de Vos <ndevos@ibm.com>

### CODE REVIEW

File: .mergify.yml [hunk 1]
```diff
@@ -34,6 +34,25 @@ queue_rules:
...
+              - "status-success=ci/centos/mini-e2e-helm/k8s-1.26"
+              - "status-success=ci/centos/mini-e2e-helm/k8s-1.27"
+              - "status-success=ci/centos/mini-e2e-helm/k8s-1.28"
+              - "status-success=ci/centos/mini-e2e/k8s-1.26"
+              - "status-success=ci/centos/mini-e2e/k8s-1.27"
+              - "status-success=ci/centos/mini-e2e/k8s-1.28"
+              - "status-success=ci/centos/upgrade-tests-cephfs"
+              - "status-success=ci/centos/upgrade-tests-rbd"
+          - and:
+              - base~=release-v3.[89]
```
Comment (Rakshith-R) [inline .mergify.yml]: let's ignore release v3.8 since it will be deprecated when 3.10 comes out ?

**Review by Rakshith-R — COMMENTED** [review]
Thanks for the changes,
left a comment regarding leaving v3.8 branch out

File: .mergify.yml (hunk 1 above)
Comment (nixpanic) [reply .mergify.yml]: sure, removed it.

**Review by Rakshith-R — APPROVED** [review]
Thanks

**Review by yati1998 — APPROVED** [review]

**Comment by nixpanic** [comment]
@mergifyio queue

**Pull request merged**